CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## [1.46.0] - 2020-21-10
+
+Breaking changes:
+- `deltachat.updateDeviceChats()` removed
+  this is now done automatically during configure
+  unless the new config-option `bot` is set deltachat-core-rust/#1957
+- `deltachat.markNoticedAllChats()` removed
+- `chat.isVerified()` is replaced by `chat.isProtected()`
+- `deltachat.createUnverifiedGroupChat(chatName)` and `deltachat.createVerifiedGroupChat(chatName)` are replaced by a single function `deltachat.createGroupChat(chatName, is_protected)`
```

Comment:
Why these two naming changes?  I don't see how they help help with things while it immediately breaks desktop.

Reply:
It is broken anyway in multiple ways, this is one of two parts, part two is an desktop pr that uses this pr.